Projects/Playground/Playground.Forms.UI/Converters/StringToUpperValueConverter.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using MvvmCross.Forms.Converters;
+
+namespace Playground.Forms.UI.Converters
+{
+    public class StringToUpperValueConverter : MvxFormsValueConverter<string, string>
```

Comment:
This is great - can you actually make use of it in XAML please too

Reply:
Yes, doing it right now, just a few minutes :-)